packages/NodeTypeResolver/Node/AttributeKey.php

```diff
@@ -221,4 +221,9 @@ final class AttributeKey
      * @var string
      */
     public const IS_BYREF_RETURN = 'is_byref_return';
+
+    /**
+     * @var string
+     */
+    public const STMT_KEY = 'stmt_key';
```

Comment:
please type your new attribute with https://github.com/rectorphp/rector/discussions/7941

Reply:
It to be filled with integer key, how to fill type?

Reply:
yeah - I guess its an int in your case. I think you need to add this line directly in https://github.com/rectorphp/phpstan-rules/blob/e50f1957565defa4e9e93371fe2fc59be9a33857/src/ReturnTypeExtension/GetAttributeReturnTypeExtension.php#L27-L35

in a dependent PR

Reply:
Ok, let's revisit after this one implemented 👍